packages/renderer/src/lib/ui/ExtensionStatus.svelte

```diff
@@ -15,7 +15,7 @@ const statusesStyle = new Map<string, connectionStatusStyle>([
     {
       bgColor: 'bg-green-500',
       txtColor: 'text-green-500',
-      label: 'ENABLED',
+      label: 'ACTIVE',
```

Comment:
Should we also update `ENABLING` to `ACTIVATING`?

Reply:
probably I'll do the changes

Reply:
I did the changes but it won't pass the CI until https://github.com/containers/podman-desktop/pull/6424 is merged